protobuf/Dockerfile

```diff
@@ -10,7 +10,7 @@ ARG GRPC_JAVA_VERSION=1.34.1
 # v1.3.2, using the version directly does not work: "tar: invalid magic"
 ARG PROTOC_GEN_GOGO_VERSION=b03c65ea87cdc3521ede29f62fe3ce239267c1bc
 ARG PROTOC_GEN_LINT_VERSION=0.2.1
-ARG GRPC_GATEWAY_VERSION=2.1.0
+ARG GRPC_GATEWAY_VERSION=1.16.0
```

Comment:
Still upgrade from 1.14.6 that was used before, but cannot use 2.1.0 with gogoproto